src/lib/getOpenApiInstance.ts

```
@@ -1,16 +1,32 @@
+  if (id === undefined) {
+    id = DEFAULT_SCHEMA
+  }
+
   if (custom?.spec) {
-    return OpenApi({ spec: custom?.spec, parsedSpec: custom?.parsedSpec })
+    return createOpenApiInstance({ spec: custom.spec })
   }
 
-  if (injected !== undefined) {
-    return injected
+  if (injected && injected.schemas) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)